app/(drawer)/(tabs)/_layout.tsx

```
@@ -0,0 +1,151 @@
+                    justifyContent: "center",
+                    alignItems: "center",
+                    paddingVertical: 10,
+                  }}
+                  onPress={onPress}
+                >
+                  {options.tabBarIcon &&
+                    options.tabBarIcon({
+                      color: isFocused ? "#FFF" : "#FFF",
+                      focused: isFocused,
+                      size: 28,
+                    })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)